packages/design-system-docs/src/pages/components/Review/Review.example.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Review } from '@design-system';
+import { Review } from '@design-system/packages/design-system';
```

Comment:
`@design-system` is an alias, so this shouldn't need to be changed. Does it break without this change?

Reply:
It does, having trouble understanding why

Reply:
@cmsgov/design-system also works

Reply:
Hmm, we're going to be running into this problem a lot then, moving forward. Maybe we should avoid converting the example files for now

Reply:
Sounds good, I'll see if I can figure out why first and if I can't will put this back to JSX